content/doc/book/blueocean/pipeline-run-details.adoc

```diff
@@ -0,0 +1,98 @@
+---
+layout: section
+title: Pipeline Run Details
+wip: true
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Pipeline Run Details
+
+The Blue Ocean Pipeline Run Details view shows the information related to
+a single Pipeline Run and allows users to edit or replay that run.
+Below is a detailed overview of the parts of the Run Details view.
+
+image:blueocean/pipeline-run-details/overview.png[Overview of the Pipeline Run Details, role=center]
+
+. *Run Status* - This icon, along with the background color of the top menu bar,
+indicates the status of this Pipeline run.
+. *Pipeline Name* - The name of this run's Pipeline.
+. *Run Number* - The id number for this Pipeline run.
+Id numbers unique for each Branch (and Pull Request) of a Pipeline.
+. *Subsection Selector* - View one of the detail <<#subsections, subsections>> for this run.
+The default is "Pipeline", show overall view the of the Pipeline flow.
+. *Re-run Pipeline* - Execute this run's Pipeline again.
+. *Edit Pipeline* - Open this run's Pipeline in the <<pipeline-editor, Pipeline Editor>>.
+. *Go to Classic* - Switch to the "Classic" UI view of the details for this run.
+. *Close Details* - This closes the Details view and returns the user to the
+<<activity, Activity view> for this Pipeline.
+. *Branch* or *Pull Request* - the branch or pull request for this run.
+. *Commit Id* - Commit id for this run.
+. *Duration* - The duration of this run.
+. *Completed Time* - How long ago the this run completed.
+. *Change Author* - Names of the authors with changes in this run.
+. *Subsection View* - Shows the information for the selected subsection.
+
+== Special cases
+
+Blue Ocean is optimized for working with Pipelines in Source Control,
+but it can handle other cases.
```

Comment:
I think this sentence could be clearer with: "it can (display|render) other projects in Jenkins as well."